msi/GitProduct.wxs

```diff
@@ -266,6 +266,8 @@
   <Fragment>
     <SetProperty Id="ExecConfigureCoreAutoCrlf" Value='"[#GitExe]" config -f "[#GfwGitConfig]" core.autocrlf [LINEENDINGS]' Sequence="execute" Before="ExecConfigureCoreAutoCrlf" />
     <CustomAction Id="ExecConfigureCoreAutoCrlf" BinaryKey="WixCA" DllEntry="WixQuietExec$(var.SixtyFourBit)" Execute="deferred" Return="ignore" Impersonate="no" />
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f /c/ProgramData/Git/config http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
This is wrong -- I reference `/c/Programdata` in the command to be executed, but what if the user's `%ROOTDRIVE%` is `D:`? I think I have a way around this. 

1. Because I'm executing this command via Bash, I should be able to use the `$ProgramData` environment variable (which is inherited from the Windows environment variable `%Programdata%`) which will resolve to the proper location.
2. Or, I could try using `/[ApplicationFolder]` and hope that the MinGW POSIX path conversion will take care of this for me. This would be better. And after all, I used this same behavior in this very command for referencing the certs themselves: `/[#ca_bundle.crt]`.

Reply:
Thanks